web/templates/user/signup3.html.eex

```diff
@@ -14,15 +14,15 @@
     <div class="form-group">
       <%= checkbox f, :terms_conditions, class: "checkbox" %>
       <%= label f, :terms_conditions do %>
-        I accept <%= link "terms and conditions", to: term_path(@conn, :index) %>
+        I accept <a href="/terms">terms and conditions.</a>
       <% end %>
       <%= error_tag f, :terms_conditions %>
     </div>
 
     <div class="mt3">
       <%= checkbox f, :privacy, class: "checkbox" %>
       <%= label f, :privacy do %>
-        I accept the terms of our <%= link "Privacy Statement", to: privacy_path(@conn, :index) %>
+        I accept the terms of our <a href="/privacy">"Privacy Statement."</a>
```

Comment:
The benefit of getting Phoenix to generate these links for us is that if we change the routes in future e.g. change the name of something. Then we won't have to also change these hard coded links, also helps with different languages etc.

This would be something like: `page_path(@conn, :show, "privacy")`